manifests/isaaclevin/presencelight/4.1.10.yaml

```diff
@@ -0,0 +1,19 @@
+Id: isaaclevin.presencelight
+Version:  4.1.10
+Name: PresenceLight
+Publisher: Isaac Levin
+License: MIT
+LicenseUrl: https://github.com/isaacrlevin/presencelight/blob/main/LICENSE
+AppMoniker: PresenceLight
+Tags: teams
+      microsoftteams
+      lifx
+      hue
+      presence
+Description: PresenceLight is a solution to broadcast your various statuses to smart lights
+Homepage: https://github.com/isaacrlevin/presencelight
+Installers:
+  - Arch: x64
+    Url: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v4.1.10/PresenceLight.4.1.10-x64.zip
```

Comment:
`.zip` format is not currently supported in winget.

Reply:
SO you need it to be msix? or appx?